.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,17 @@
+name: Test repository
```

Comment:
delete file in favor of unifying in the other workflow and execute the tests only one

Reply:
Not sure this is a good idea. Having an action only trigger on the right conditions seems better to me. You are unlikely to have a run away event where you end up with 5 or more actions.